pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -567,11 +571,6 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 		return nil
 	}
 
-	// If we have seen this generation then skip
-	if cfg.Status.ObservedGeneration >= cfg.Generation {
```

Comment:
this is to make sure that we can resync a failed status, right?

Reply:
yes